### DISCUSSION TITLE
Infinispan would like to join Commonhaus

### DISCUSSION THREAD

**Discussion by tristantarrant** [commonhaus/foundation#215] [Joining Commonhaus]
### Project information

- **Project name**: Infinispan
- **Project website**: https://infinispan.org
- **Code repository**: https://github.com/infinispan/infinispan
- **License**: Apache License, Version 2.0

### Do you have authority to represent this project?

- [X] I own or maintain this project

### Describe your project

Infinispan is an open-source in-memory database that can hold nearly any type of data, from plain-text to structured objects. It is implemented in Java and can be used either embedded or in client/server mode, accessed using either a native, high-performance protocol, or any HTTP, Redis or Memcached clients. Data can be queried via full-text and vector search capabilities. Infinispan can be elastically scaled offering high availability, fault tolerance, and the ability to replicate between multiple geographic sites.

### Motivation for joining Commonhaus

Infinispan is a mature project that continues to evolve. We want to foster that evolution by:
* Growing our community, both via individual contributors and via adoption by other projects
* Being more open about our roadmap, our designs and our discussions
* CommonHaus, with its community-first governance model, is an ideal vehicle for Infinispan's intentions.

### Things that must be done (bootstrap phase)

- [x] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting).
- [x] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [x] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)

**Comment by tristantarrant** [comment]
Bootstrapping PR https://github.com/commonhaus/foundation/pull/216

**Comment by gavinking** [comment]
Nice!

**Comment by ebullient** [comment]
Welcome to Commonhaus! 🚀
